### PR TITLE
don't bring in the whole assert library

### PR DESCRIPTION
**Pull request by calvinmetcalf**
I grabbed the assert function you used in bn.js, considering you only use assert() assert by itself is pretty big PLUS it brings in utils

### CODE REVIEW

File: lib/elliptic/utils.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
-var assert = require('assert');
 var bn = require('bn.js');
 
 var utils = exports;
-
```
Comment (indutny) [inline lib/elliptic/utils.js]: Please do not remove newline here.

File: lib/elliptic/utils.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+utils.assert = function assert(val, msg) {
+  if (!val)
+    throw new Error(msg || 'Assertion failed');
+};
```
Comment (indutny) [inline lib/elliptic/utils.js]: Please add a newline after this line.

File: lib/elliptic/hmac-drbg.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-var assert = require('assert');
```
Comment (indutny) [inline lib/elliptic/hmac-drbg.js]: I guess there is no need in this newline, right?

File: lib/elliptic/curve/base.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
-var assert = require('assert');
 var bn = require('bn.js');
 var elliptic = require('../../elliptic');
 
+
```
Comment (indutny) [inline lib/elliptic/curve/base.js]: Superfluous newline.

**Comment by indutny** [comment]
Thanks! Generally LGTM, except some style nits.

**Comment by calvinmetcalf** [comment]
fixed!

**Comment by indutny** [comment]
Landed!

**Pull request closed without merging**